### ISSUE TITLE
Link text is not visible in the tree when the row is selected in Default Light themes

### ISSUE THREAD

**Issue by sandy081** [microsoft/vscode#130920]
Steps to Reproduce:

1. Install `davidanson.vscode-markdownlint` extension
2. Create a sample markdown file and enter text `## Hello`
3. Open Problems pane and see there are problems with links

![image](https://user-images.githubusercontent.com/10746682/129596632-ad9c5593-e547-4dcc-9496-cfffcd8cd127.png)

4. Select one of the problems

🐛 Link text is not visible 

![image](https://user-images.githubusercontent.com/10746682/129596605-807338a5-5785-42f1-aba3-619efcca22dc.png)

It seems list selection color and link text color are not going well together. This happens only with out of the box Light and Light+ themes. Other themes are good.

**Comment by misolori** [comment]
I believe your recent change removed this styling:

https://github.com/microsoft/vscode/commit/2ea4dcda7e218e7d8294711688b154b03287f4bb#diff-8fdf5a358bf7c139cd69884cbb4da7dd9203d4e4e6685b0c759594e10768e39aL142-L144

Re-added this css rule and scoped it to the list focus state:

<img width="740" alt="CleanShot 2021-08-16 at 10 15 29@2x" src="https://user-images.githubusercontent.com/35271042/129603192-c7b578e9-2491-43bc-ab0f-ba25084b3633.png">

<img width="604" alt="CleanShot 2021-08-16 at 10 17 33@2x" src="https://user-images.githubusercontent.com/35271042/129603437-31a358ca-c8bf-43b8-9585-1ef0a8691c73.png">
